src/test/java/microservices/userServiceBasic/UserDtoServiceBasicApplicationTests.java

```diff
@@ -7,7 +7,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class UserServiceBasicApplicationTests {
+public class UserDtoServiceBasicApplicationTests {
```

Comment:
Should be renamed to `UserServiceBasicApplicationTests ` as it was

Reply:
These are different, there is one for each.